src/spv/spv_wrapper.cpp

```diff
@@ -489,6 +500,17 @@ void CSpvWrapper::UpdateTx(uint256 const & hash, uint32_t blockHeight, uint32_t
     }
 }
 
+uint32_t CSpvWrapper::ReadTxTimestamp(uint256 const & hash)
+{
+    std::pair<char, uint256> const key{std::make_pair(DB_SPVTXS, hash)};
+    std::pair<TBytes, std::pair<uint32_t, uint32_t> > txrec;
```

Comment:
Can use `db_tx_rec` instead of `std::pair<TBytes, std::pair<uint32_t, uint32_t> >`

Reply:
So we can...

Reply:
https://github.com/DeFiCh/ain/commit/9bc92060574676aae85ddab7d0dbd845d66e9b39